src/engine/Engine.cpp

```diff
@@ -773,7 +773,7 @@ void Engine::removeRedundantEquations( const double *constraintMatrix )
     analyzer->analyze( constraintMatrix, m, n );
 
     ENGINE_LOG( Stringf( "Number of redundant rows: %u out of %u",
-                  analyzer->getRedundantRows().size(), m ).ascii() );
+                         analyzer->getRedundantRows().size(), m ).ascii() );
```

Comment:
remove spaces

Reply:
The new spaces? It was not properly indented before, now it is directly below the "